### PR TITLE
Updated Version: account creation/ cookies/ overall structure

### PR DESCRIPTION
**Pull request by Aayush-Srikanth**
-Deleted logResponseDetails (No Usages)
- Base URI already exists --> no use for URL string, changed to endpoint

BaseTest:
- Deleted getRequest test, was not required
- Added Before/ After statements (empty)

CreateAccountPage:
- Utilised get status code and get Gpath body and moved assertions to the createNewAccountTest
- Renamed method 'createAcc' --> 'createNewAccount'

CreateAccTest:
- Added constructor initializing createAccountPage/ homePage
- createNewAccountTest --> added status code assertion/ page element assertion

HomePage:
- Altered getHomepage method --> removed redundant string
- Added method getLoggedInUsername --> returns username from Gpath

### CODE REVIEW

File: src/test/java/PageObjects/BasePage.java [hunk 1]
```diff
@@ -15,51 +14,43 @@ abstract class BasePage {
...
 
-    public void logResponseDetails() {
-        System.out.println("Status Code = " + getStatusCode());
-        System.out.println("Cookies = " + getCookies());
-    }
-
-    public Cookies getCookies() {
-        return response.getDetailedCookies();
-    }
+    //public Response sendPostRequest()
```
Comment (Leosaw) [inline src/test/java/PageObjects/BasePage.java:46]: I take it that's for later use?

File: src/test/java/PageObjects/CreateAccountPage.java [hunk 2]
```diff
@@ -1,17 +1,14 @@
...
 import java.util.Map;
 
-
 public class CreateAccountPage extends BasePage {
 
     private String generateEmail = RandomStringUtils.randomAlphabetic(7);
 
-    public void createAcc() {
+    public void createNewAccount() {
         //Base URI defined in Base Page
```
Comment (Leosaw) [inline src/test/java/PageObjects/CreateAccountPage.java:12]: We can clean that up later

File: src/test/java/PageObjects/HomePage.java [hunk 3]
```diff
@@ -2,9 +2,11 @@
 
 public class HomePage extends BasePage{
 
-
     public void getHomePage() {
-        String url = "http://3.11.77.136/index.php";
-        response = sendGetRequest(url);
```
Comment (Leosaw) [inline src/test/java/PageObjects/HomePage.java:8]: Could peeps remind me I need to ask @matthewmccormick matt about this later? It is to do with possibly making a method to get and page on the website and placing it in basepage or having multiple methods in different PageObjects.

File: src/test/java/Tests/CreateAccTest.java [hunk 4]
```diff
@@ -2,20 +2,30 @@
...
+    CreateAccountPage createAccountPage;
+
+    public CreateAccTest() {
+        createAccountPage = new CreateAccountPage();
+        homePage = new HomePage();
+    }
+
     @Test
-    public void Test2(){
+    public void getHomepageTest() {
```
Comment (Leosaw) [inline src/test/java/Tests/CreateAccTest.java:19]: We possibly need to move that into another class for consistency/conciseness.

**Review by Leosaw — APPROVED** [review]
Awesome @Aayush-Srikanth

**Review by jacktegart — APPROVED** [review]

**Review by SerineBetraoui — APPROVED** [review]

**Review by SuzyLady — APPROVED** [review]

File: src/test/java/PageObjects/BasePage.java (hunk 1 above)
Comment (Aayush-Srikanth) [reply src/test/java/PageObjects/BasePage.java:46]: Yea still need to complete this

**Pull request merged**